.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@frrist
```

Comment:
is this the same as this?
```suggestion
*	@frrist
```

Reply:
:shrug: github says this current file is valid? So I assume it's fine until more folks wanna join as owners?